open-api/rest-catalog-open-api.yaml

```diff
@@ -1752,6 +2223,26 @@ components:
               items:
                 type: string
 
+    AddViewVersionUpdate:
+      allOf:
+        - $ref: '#/components/schemas/BaseUpdate'
+        - type: object
+          required:
+            - view-version
+          properties:
+            view-version:
+              $ref: '#/components/schemas/ViewVersion'
+
+    SetCurrentViewVersionUpdate:
+      allOf:
+        - $ref: '#/components/schemas/BaseUpdate'
+        - type: object
+          required:
+            - view-version-id
+          properties:
+            view-version-id:
+              type: integer
+
     TableUpdate:
```

Comment:
We might want to split this out into `ViewUpdate` to keep the operations specific to the endpoints.

Reply:
makes sense, I've updated this in the latest commit